models.go

```diff
@@ -34,12 +34,16 @@ type TestRun struct {
     CreatedAt       time.Time       `json:"created_at"`
 }
 
-
 type Browser struct {
     InitiallyLoaded bool            `json:"initially_loaded"`
+    CurrentlyRun    bool            `json:"currently_run"`
     BrowserName     string          `json:"browser_name"`
     BrowserVersion  string          `json:"browser_version"`
     OSName          string          `json:"os_name"`
     OSVersion       string          `json:"os_version"`
     Sauce           bool            `json:"sauce"`
 }
+
+type Token struct {
+    Secret          string          `json:"secret"`
+}
```

Comment:
This is used for authenticating `TestRun` uploads. `POST /test-runs` pulls a `Token` with a hard-coded key out of the datastore and checks it against the URL param supplied in `run/run.py`.